openreview/venue/webfield/areachairsWebfield.js

```diff
@@ -1,56 +1,56 @@
+  browseInvitations.push(domain.content.reviewers_affinity_score_id?.value)
+  browseProposedInvitations.push(domain.content.reviewers_affinity_score_id?.value)
 }
 
 if (domain.content.bid_name?.value) {
-  browseInvitations.push(`${reviewerGroup}/-/${domain.content.bid_name.value}`)
-  browseProposedInvitations.push(`${reviewerGroup}/-/${domain.content.bid_name.value}`)
+  browseInvitations.push(`${reviewerGroup}/-/${domain.content.bid_name?.value}`)
+  browseProposedInvitations.push(`${reviewerGroup}/-/${domain.content.bid_name?.value}`)
 }
 
 if (domain.content.reviewers_custom_max_papers_id?.value) {
-  browseInvitations.push(`${domain.content.reviewers_custom_max_papers_id.value},head:ignore`)
-  browseProposedInvitations.push(`${domain.content.reviewers_custom_max_papers_id.value},head:ignore`)
+  browseInvitations.push(`${domain.content.reviewers_custom_max_papers_id?.value},head:ignore`)
+  browseProposedInvitations.push(`${domain.content.reviewers_custom_max_papers_id?.value},head:ignore`)
 }
 
-const otherParams = `&hide=${domain.content.reviewers_conflict_id.value}&maxColumns=2&version=2&referrer=[AC%20Console](/group?id=${areaChairsId})`
+const otherParams = `&hide=${domain.content.reviewers_conflict_id?.value}&maxColumns=2&version=2&referrer=[AC%20Console](/group?id=${areaChairsId})`
 
 return {
   component: 'AreaChairConsole',
   version: 1,
   properties: {
     header: {
       title: 'Area Chair Console',
-      instructions: `<p class="dark">This page provides information and status updates for the ${domain.content.subtitle.value}. It will be regularly updated as the conference progresses, so please check back frequently.</p>`,
+      instructions: `<p class="dark">This page provides information and status updates for the ${domain.content.subtitle?.value}. It will be regularly updated as the conference progresses, so please check back frequently.</p>`,
     },
     apiVersion: 2,
     venueId: domain.id,
     reviewerAssignment: {
       showEdgeBrowserUrl: domain.content.enable_reviewers_reassignment?.value,
       proposedAssignmentTitle: reviewerAssignmentTitle,
-      edgeBrowserProposedUrl: `/edges/browse?start=${startParam}&traverse=${traverseProposedParam}&edit=${domain.content.reviewers_proposed_assignment_id.value},label:${reviewerAssignmentTitle};${domain.content.reviewers_invite_assignment_id.value}&browse=${browseProposedInvitations.join(';')}${otherParams}`,
-      edgeBrowserDeployedUrl: `/edges/browse?start=${startParam}&traverse=${traverseParam}&edit=${domain.content.reviewers_invite_assignment_id.value}&browse=${browseInvitations.join(';')}${otherParams}`,
+      edgeBrowserProposedUrl: `/edges/browse?start=${startParam}&traverse=${traverseProposedParam}&edit=${domain.content.reviewers_proposed_assignment_id?.value},label:${reviewerAssignmentTitle};${domain.content.reviewers_invite_assignment_id?.value}&browse=${browseProposedInvitations.join(';')}${otherParams}`,
```

Comment:
if the .value may be undefined, i think the &edit should be removed too instead of passing &edit=undefined,label=...